docs/guides/lint.md

```diff
@@ -3,75 +3,237 @@ redirectFrom:
   - /docs/cli/configuration/lint/
 ---
 
-# Lint configuration
```

Comment:
We start each article with a heading level 1. This helps our people understand what topic they are on, and helps search engines know about about the topic too.